lib/TH/generic/THTensor.c

```diff
@@ -510,6 +510,28 @@ void THTensor_(squeeze1d)(THTensor *self, THTensor *src, int dimension)
   }
 }
 
+int THTensor_(isTransposed)(const THTensor *self)
```

Comment:
Well, I tried `t:select(2,0):expand(10,32)`, but it appears that torch does not allow to do that.

> stdin:1:` bad argument #2 to 'select' (out of range at